Sprintfinity3902/Sprintfinity3902.csproj

```diff
@@ -10,6 +10,11 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
     <ApplicationIcon>Icon.ico</ApplicationIcon>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Hud\**" />
+    <EmbeddedResource Remove="Hud\**" />
+    <None Remove="Hud\**" />
+  </ItemGroup>
```

Comment:
This is strange to me, I just have no idea why this would have changed.

Reply:
When I tried to make a Hud folder initially it yelled at me that I couldn't use that name? Perhaps it has something to do with this... but I am honestly not sure how it happened.